homeassistant/components/yolink/binary_sensor.py

```diff
@@ -43,19 +43,22 @@ class YoLinkBinarySensorEntityDescription(BinarySensorEntityDescription):
 
 SENSOR_TYPES: tuple[YoLinkBinarySensorEntityDescription, ...] = (
     YoLinkBinarySensorEntityDescription(
+        key="doorState",
```

Comment:
Let's use snake_case if it's just a name we decide here.
```suggestion
        key="door_state",
```